service/src/main/java/bio/terra/cbas/dao/RunDao.java

```diff
@@ -39,9 +39,29 @@ public List<Run> getRuns() {
   }
 
   public int updateRunStatus(Run run, CbasRunStatus newStatus) {
-    String sql = "UPDATE run SET status = :status WHERE id = :id";
+    OffsetDateTime currentTimestamp = OffsetDateTime.now();
+    String sql =
+        "UPDATE run SET status = :status, last_modified_timestamp =:lastModifiedTimestamp, last_polled_timestamp = :lastPolledTimestamp  WHERE id = :id";
+    return jdbcTemplate.update(
+        sql,
+        new MapSqlParameterSource(
+            Map.of(
+                "id",
+                run.id(),
+                "status",
+                newStatus.toString(),
+                "lastModifiedTimestamp",
+                currentTimestamp,
+                "lastPolledTimestamp",
+                currentTimestamp)));
+  }
+
+  public int updateLastPolledTimestamp(UUID runID) {
```

Comment:
TOL: Since these will _always_ be updated for every run, is it possible to update the whole set of runs together in bulk rather than one at a time?

Reply:
So when we call `updateRunStatus` we only update the `last_modified_timestamp`. And at the end in SmartPoller, before we return the list of Runs, we do a bulk update for `last_polled_timestamp`?

Reply:
Something like that. Not necessary for this PR, we could keep it in our back pockets in case things are scaling badly in the future...